storage/local/index.ts

```diff
@@ -74,16 +57,45 @@ export async function getLocalStorage() {
           .map(([id, a]) => ({ id, ...a }))
           .filter((a) => a.chainId === n.chainId)[0]?.id
       }
```

Comment:
The `account` will be `null` because it hasn't been declared yet, resulting in a "ReferenceError: Cannot access 'account' before initialization" error.

Removing these lines should work fine:

```typescript
      if (!targetNetwork.accountActive) {
        targetNetwork.accountActive = Object.entries(account)
          .map(([id, a]) => ({ id, ...a }))
          .filter((a) => a.chainId === n.chainId)[0]?.id
      }
```